apps/extension/background/messages/posthog.ts

```diff
@@ -1,4 +1,4 @@
-import { posthog } from "posthog-js"
+import { posthog } from "posthog-js/dist/module.no-external"
```

Comment:
why is this necessary? Just for context

Reply:
Posthog loads external scripts at runtime which is against Chrome web store policies